http/codegen/typedef.go

```diff
@@ -59,7 +59,7 @@ func goTypeDef(scope *codegen.NameScope, att *expr.AttributeExpr, ptr, useDefaul
 				fn = codegen.GoifyAtt(at, name, true)
 				tdef = goTypeDef(scope, at, ptr, useDefault)
 				if expr.IsPrimitive(at.Type) {
-					if ptr || mat.IsPrimitivePointer(name, useDefault) {
+					if (ptr || mat.IsPrimitivePointer(name, useDefault)) && at.Type != expr.Bytes && at.Type != expr.Any {
```

Comment:
I'm wondering whether `IsPrimitivePointer` shouldn't include these checks? maybe it can't because we need to differentiate in other places.

Reply:
The problem here is we force the generation of pointers using `ptr` param. Even if `IsPrimitivePointer` is `false` we generate the type as pointer if `ptr` is true. We need to create exceptions for the `Byte` and `Any` type like above.

Reply:
Makes sense, thanks for the explanation.